tests/example_integration.rs

```diff
@@ -0,0 +1,194 @@
+use std::net::{IpAddr, Ipv4Addr, SocketAddr};
+use serial_test::serial;
+use bevy::MinimalPlugins;
+use bevy::core::{Pod, Zeroable};
+use bevy::input::{ButtonState, Input, InputPlugin};
+use bevy::input::keyboard::KeyboardInput;
+use bevy::prelude::*;
+use bevy_ggrs::{GGRSPlugin, GGRSSchedule, PlayerInputs, Rollback, RollbackIdProvider, Session};
+use ggrs::{Config, SessionBuilder, PlayerType, UdpNonBlockingSocket, PlayerHandle, P2PSession};
+
+#[test]
+#[serial]
+fn it_syncs_rollback_resources() -> Result<(), Box<dyn std::error::Error>> {
+    let (player1, player2) = create_players();
+    let session1 = start_session(&player1, &player2)?;
+    let mut app1 = create_app::<GGRSConfig>(session1);
+    let session2 = start_session(&player2, &player1)?;
+    let mut app2 = create_app::<GGRSConfig>(session2);
+
+    for _ in 0..50 {
+        app1.update();
+        app2.update();
```

Comment:
This one doesn't actually do any syncing though? Perhaps we should mutate a resource based on player input to make sure something is actually happening that require resource syncing to work?

Reply:
Yeah good catch, this was my first exploratory test, and it has a misleading name.

This one just ended up testing if the rollback systems run individually in each application.